src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/User.kt

```diff
@@ -1,5 +1,14 @@
 package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation
 
 data class User(
+  val email: String?,
+  val username: String?,
+  val staffIdentifier: Long?,
+  val staff: StaffHuman?,
   val teams: List<Detail>,
 )
+
+data class StaffHuman(
```

Comment:
how about StaffHuman->StaffName?

Reply:
Think this is what it is called in the CommunityApi?

Reply:
yes, the names are a bit rubbish in CommunityApi, also `User` refers to `Staff` which is really confusing as there is another entity called `User`!  Thought it better to match what the swagger exposes for consistency. 

Hopefully the probation integration team will replace this with a bespoke CVL API at some point.